locales/ara.json

```diff
@@ -159,7 +159,8 @@
   },
   "IssuersScreen": {
     "title": "أضف بطاقة جديدة",
-    "header": "يرجى اختيار جهة الإصدار المفضلة لديك من الخيارات أدناه لإضافة بطاقة جديدة.",
+    "addCardDescription": "يرجى اختيار جهة الإصدار المفضلة لديك من الخيارات أدناه لإضافة بطاقة جديدة.",
```

Comment:
please rename this key from addCardDescription to **SubHeader or description.** in all locales.

Reply:
If I tries to change text key as "subHeader" also need to change its testID into same name. @swatigoel mentioned that don't change testID. So I remained it as same.